src/Matroids/ChowRings.jl

```diff
@@ -0,0 +1,95 @@
+export chow_ring, select
+
+function chow_ring(M::Matroid, extended=false)
+	Flats = flats(M)
+	sizeFlats = size(Flats)[1]
+	n = size(M.groundset)[1]
+	if(!is_loopless(M))
+		throw("Matroid has loops")
+	end
```

Comment:
More idiomatic would be something like this (also: don't throw strings):
```suggestion
	is_loopless(M) || error("Matroid has loops")
```